spec/lib/license_finder/package_managers/go_dep_spec.rb

```diff
@@ -140,14 +140,10 @@ module LicenseFinder
           end
         end
 
-        context 'when dependencies are not vendored' do
```

Comment:
So, does this mean we no longer have a case where dependencies will not be vendered?